decidim-core/db/migrate/20200327082257_migrate_newsletters_to_templates.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+class MigrateNewslettersToTemplates < ActiveRecord::Migration[5.2]
+  class ContentBlock < ApplicationRecord
+    self.table_name = :decidim_content_blocks
+  end
+
+  class Newsletter < ApplicationRecord
+    self.table_name = :decidim_newsletters
+  end
+
+  def change
+    remove_index :decidim_content_blocks, name: "idx_dcdm_content_blocks_uniq_org_id_scope_manifest_name"
```

Comment:
A unique index on scope_id? I should ignore the "null" values then (because the homepage blocks set it as null)... Thoughts?

Reply:
I was assuming that a unique index with a nullable column would allow only one row with the same values and a null value in that column. That would allow us to keep the previous behaviour at the database level, raising an error when trying to add the same block twice to the homepage. But [I was wrong](https://www.postgresql.org/docs/9.0/indexes-unique.html), so we probably should leave it removed, as you did.